scripts/00150-6lackHat.py

```diff
@@ -0,0 +1,10 @@
+def Hello(full_name, ID, language, email):
+    return "Hello World, this is {} with HNG ID {} using {} for stage 2 task".format(full_name, ID, language,email)
```

Comment:
Please check this. Your email should come after the word 'task'. Right now, that is not done

Reply:
done